### PR TITLE
feat: add addtional check in Wordle initialization

### PR DESCRIPTION
**Pull request by wodeni**
# Description

The original Wordle game allows an `answer` parameter for testing. Instead of allowing arbitrary words as a parameter, I added an additional check to make sure `answer` is actually in the dictionary. 

# Open questions

* My tests are failing, not really sure why tho. Can you help me?

### CODE REVIEW

**Review by lviornery — CHANGES_REQUESTED** [review]
'animo' is used as a set answer in the test suite but was removed from word list. Change tests or restore word list to previous state.

File: recitation-8/words.json [hunk 1]
```diff
@@ -13231,7 +13231,6 @@
     "animists": 1,
     "animize": 1,
     "animized": 1,
-    "animo": 1,
```
Comment (MichaelHilton) [inline recitation-8/words.json:13234]: hmmm, this line seems to be a problem.

**Review by darrcheng — CHANGES_REQUESTED** [review]
You deleted amino but are testing for it. You should add amino back into the dictionary.

File: recitation-8/wordle_test.py [hunk 2]
```diff
@@ -121,4 +121,9 @@ def test_gid_pizza():
 def test_gid_pizz():
     assert not(guess_in_dict('pizz', {'pizza': None}))
 
-# TODO write at least one test for `pick_word`
+
+def test_setup_anwser_animo():
+    dict_path = './words.json'
+    dictionary = load_words(dict_path)
+    wordle = WordleGame(dictionary, answer="animo")
+    assert wordle.answer_word == "animo"
```
Comment (dcasalnu) [inline recitation-8/wordle_test.py:129]: You are trying to test a word that you removed from the dictionary

File: recitation-8/words.json (hunk 1 above)
Comment (MichaelHilton) [reply recitation-8/words.json:13234]: please change the word list to add this word back in.

File: recitation-8/wordle_test.py (hunk 2 above)
Comment (kathjordan) [reply recitation-8/wordle_test.py:129]: This looks like the problem to me, too

File: recitation-8/words.json (hunk 1 above)
Comment (joshsunshine) [reply recitation-8/words.json:13234]: @MichaelHilton I think the problem is the test. Animo is a stupid word.

**Comment by jackj26** [comment]
Animo was removed from the dictionary, but it was part of the test document. May need to either add back animo or test a different word.

**Review by nikolechetty — CHANGES_REQUESTED** [review]
Add "animo" back to word.json

File: recitation-8/words.json (hunk 1 above)
Comment (c-troller) [reply recitation-8/words.json:13234]: Need to add word back into dictionary for the test.

Comment (morganchendev) [reply recitation-8/words.json:13234]: You took out this word "animo".

**Pull request closed without merging**